index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@500&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+    <header>
+        <nav class="nav-bar">
+            <img id="shapes" src="/images/shapes.svg" alt="">
+            <img id="hamburger" src="/images/ic_menu.svg" alt="">
```

Comment:
I think you did a great job adding these images to your project however the images are not rendering. Please edit the src attribute of your images. :pray: :pray:  You could try this `src="./images/image.svg"`. Kindly pay attention to the `.` at the beginning of the path. Maybe in a local environment, everything is working fine but when you deploy your project the images are not rendering.
![image](https://user-images.githubusercontent.com/98363075/198164985-5c194708-47a4-4019-8113-26c55a9174e6.png)